src/Forms/Form.php

```diff
@@ -884,7 +885,7 @@ public function getAttributesHTML($attrs = null)
 
         // If we need to disable cache, do it
         if ($needsCacheDisabled) {
-            HTTP::set_cache_age(0);
+            HTTPCacheControlMiddleware::singleton()->disableCache();
```

Comment:
Good fix but really odd that we do this in getAttributesHTML lol.

Reply:
Yeah, which makes me worried - maybe it should really be in `forTemplate` ?

Reply:
That would probably be tidier.